packages/zeebe-element-templates-json-schema/src/defs/template.json

```diff
@@ -372,6 +372,67 @@
           "properties"
         ]
       }
+    },
+    {
+      "if": {
```

Comment:
This piece means: "If there is a property with binding `zeebe:taskDefinition`, either element type must be `bpmn:ScriptTask`, or the template is allowed for `bpmn:ScriptTask` only.

Reply:
We want to disallow task definition if a script is used. Element type constraints are for `zeebe:Script` only as it's the only element type which can contain that extension.

Reply:
sorry my comment was posted before refreshing the page, forget it until I don't go through your latest comments